common/app/experiments/Experiments.scala

```diff
@@ -30,3 +30,12 @@ object StickyVideos
       sellByDate = LocalDate.of(2022, 6, 2),
       participationGroup = Perc0C,
     )
+
+object SlideshowCaptions
```

Comment:
Does this need to be added to the above `allExperiments` list?

Reply:
It does we can do that in your PR.